ProjectMars/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,275 @@
+        private IWebElement editTextBox => webDriver.FindElement(By.XPath("//input[@placeholder='Add Language']"));
+        private IWebElement updateButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td/div/span/input[1]"));
+        private IWebElement newEditLanguage => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[1]"));
+        private IWebElement newEditLevel => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[2]"));
+        private IWebElement deleteButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]/i"));
+
+
+        public void GotoLanguageTab(IWebDriver driver)
+        {
+            Thread.Sleep(2000);
+            //wait.Waittobeclickable(driver, "Xpath", "//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[1]", 5);         
+            languageTab.Click();
+        }
+        public void GotoSkillsTab(IWebDriver driver)
+        {
+            //wait.Waittobeclickable(driver, "Xpath", "//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[2]", 2);
+            Thread.Sleep(2000);
+            skillTab.Click();
+        }
+        public void ResetLanguageRow(IWebDriver driver)
+        {
+            int rowCount = driver.FindElements(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr")).Count;
+            
+            for (int i = 0; i < rowCount; i++)
+            {
+                Thread.Sleep(2000);
+                IWebElement deleteButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]"));    
+                Thread.Sleep(2000);
+                deleteButton.Click();
+            }
+        }
+        public void ResetSkillRow(IWebDriver driver)
+        {
+            int rowCount = driver.FindElements(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody/tr")).Count;
+            for (int i = 0; i < rowCount; i++)
+            {
+                Thread.Sleep(2000);
+                IWebElement deleteButton = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]"));
```

Comment:
please move this into the pom pattern design, also do this all web elements that shoud moved out of action methods